### PR TITLE
Pylint

### PR DESCRIPTION
**Pull request by tadeboro**
CHanges in this PR introduce pylint development aid for helping keep our codebase consistent to at least some extent.

I used default pylint settings that enforce PEP8 style with some additional checks for lousy variable names. Current style settings are not meant to be the final version of the style guide, but I guess they are as good as any other for the time being.

Feel free to suggest additions and changes (now or anytime in the future).

/cc @aljazkosir

### CODE REVIEW

File: pylintrc [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+spelling-private-dict-file=
+spelling-store-unknown-words=no
+
+
+[FORMAT]
+expected-line-ending-format=
+ignore-long-lines=^\s*(# )?<?https?://\S+>?$
+indent-after-paren=4
+indent-string='    '
+max-line-length=80
```
Comment (miha-plesko) [inline pylintrc]: TBH I prefer a little wider lines than breaking too often. How does 120 chars sound to you?

Comment (matejart) [reply pylintrc]: In my opinion, anything beyond 80 columns is just too wi|

Comment (miha-plesko) [reply pylintrc]: Okay, merging

Comment (tadeboro) [reply pylintrc]: I usually stick to 80 chars when writing code or documents, but this is just my personal preference (I work with files in vim's vertical splits, and 80 character lines allow me to cram three splits + console on the screen).

Another popular choice besides PEP8 is [black style][black] that is used by the black code formatter. This style uses 88 chars from purely pragmatic reasons.

   [black]: https://black.readthedocs.io/en/stable/the_black_code_style.html

But I am fine with almost anything as long as we keep it consistent.

**Pull request merged**